addon/styles/addon.scss

```diff
@@ -1,38 +1,63 @@
 @import 'node_modules/ember-frost-css-core/scss/frost-core';
 @import 'node_modules/ember-frost-theme/scss/frost-theme';
 
-$border: 1px solid $frost-color-lgrey-5;
+$border: 1px solid $frost-color-grey-6;
 $color: $frost-color-grey-2;
 $error-border: 1px solid $frost-color-danger;
 $error-margin: 1px; // Note: margin + border width equals border width of focus state
 $focus-border: 2px solid $frost-color-blue-1;
 $focus-margin: 0;
+$disabled-border: 1px solid $frost-color-lgrey-1;
+$disabled-bg: $frost-color-lgrey-3;
 $font-size: $frost-font-s;
-$height: 30px;
+$height: 35px;
 $margin: 1px; // Note: margin + border width equals border width of focus state
 $padding: 0 6px;
 
 .frost-text {
-  border: $border;
-  color: $color;
-  font-size: $font-size;
-  height: $height;
-  margin: $margin;
-  outline: none;
-  padding: $padding;
-
-  &:focus {
-    border: $focus-border;
-    margin: $focus-margin;
+  &.error {
+    > input {
+      border: $error-border;
+      margin: $error-margin;
+    }
   }
 
-  &:read-only {
-    border: none;
-    margin: $focus-margin;
+  > input {
+    border: $border;
+    color: $color;
+    font-size: $font-size;
+    height: $height;
+    margin: $margin;
+    outline: none;
+    padding: $padding;
+
+    &:focus {
+      border: $focus-border;
+      margin: $focus-margin;
+    }
+
+    &:read-only {
+      border: none;
+      margin: $focus-margin;
+    }
+
+    &:disabled {
+      border: $disabled-border;
+      background-color: $disabled-bg;
+    }
+
+    &:hover:enabled:read-write{
+      border: $focus-border;
+    }
   }
 
-  &.error {
-    border: $error-border;
-    margin: $error-margin;
+  .close {
+    position: relative;
```

Comment:
I'm a little concerned about this for two reasons:
- What happens to any content on the right side of this if the close appears/disappears?  
- What happens if the text flows under the X?

Can we position it relative inside the input instead?